addons/sale_coupon/tests/test_program_numbers.py

```diff
@@ -1146,3 +1146,53 @@ def test_program_modify_reward_line_qty(self):
         self.assertEqual(len(order.order_line), 2, 'The order must contain 2 order lines: 1x Product F and 1x 5$ discount')
         self.assertEqual(order.amount_total, 190.0, 'The price must be 190.0 since there is now 2x 5$ discount and 2x Product F')
         self.assertEqual(order.order_line.filtered(lambda x: x.is_reward_line).price_unit, -5, 'The discount unit price should still be -5 after the quantity was manually changed')
+
+    def test_program_multiple_promotions(self):
+        testprod = self.env['product.product'].create({
+            'name': 'testprod',
+            'lst_price': 100.0,
+        })
+
+        test10_unique = self.env['sale.coupon.program'].create({
+            'name': '10% discount on first order',
+            'promo_code_usage': 'no_code_needed',
+            'program_type': 'promotion_program',
+            'discount_type': 'percentage',
+            'discount_percentage': 10.0,
+            'maximum_use_number': 1,
+        })
+        test05 = self.env['sale.coupon.program'].create({
+            'name': '5% discount',
+            'promo_code_usage': 'no_code_needed',
+            'program_type': 'promotion_program',
+            'discount_type': 'percentage',
+            'discount_percentage': 5.0,
+        })
+
+        #check that 10% is applied at first order
+        order = self.env['sale.order'].create({'partner_id': self.steve.id})
+        order_line = self.env['sale.order.line'].create({
+            'product_id': testprod.id,
+            'name': 'testprod',
+            'product_uom_qty': 1.0,
+            'price_unit': 100.0,
+            'order_id': order.id,
+            'tax_id': False,
```

Comment:
```suggestion
            'order_id': order.id,
```
I don't think those 2 are needed